includes/class-indieauth-authorization-endpoint.php

```diff
@@ -145,7 +145,7 @@ public static function scopes( $scope = 'all' ) {
 	public static function scope_list( $scopes ) {
 		if ( ! empty( $scopes ) ) {
 			foreach ( $scopes as $s ) {
-				printf( '<li><input type="checkbox" name="scope[]" value="%1$s" %2$s /><strong>%1$s</strong> - %3$s</li>', $s, checked( true, true, false ), self::scopes( $s ) );
+				printf( '<li><input type="checkbox" name="scope[]" value="%1$s" %2$s /><strong>%1$s</strong> - %3$s</li>', $s, checked( true, true, false ), esc_html( self::scopes( $s ) ) ); // phpcs:ignore
```

Comment:
This doesn't need to be phpcs:ignored. It can be escaped like this without warning:
```suggestion
				echo wp_kses(
					sprintf( '<li><input type="checkbox" name="scope[]" value="%1$s" %2$s /><strong>%1$s</strong> - %3$s</li>', $s, checked( true, true, false ), self::scopes( $s ) ),
					array(
						'li'     => array(),
						'strong' => array(),
						'input'  => array(
							'type'    => array(),
							'name'    => array(),
							'value'   => array(),
							'checked' => array(),
						),
					)
				);
```